CMakeLists.txt

```diff
@@ -123,6 +123,13 @@ ENDIF()
 
 set(CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/modules/" ${CMAKE_MODULE_PATH})
 
+# Set MSVC directory
```

Comment:
I think you might want to actually split this file's change out into it's own PR as well.